misc/de-metas-common/de-metas-common-rest_api/src/main/java/de/metas/common/rest_api/v2/project/workorder/JsonWorkOrderProjectResponse.java

```diff
@@ -71,7 +71,7 @@
 	LocalDate dateFinish;
 
 	@Nullable
-	JsonMetasfreshId bPartnerId;
+	JsonMetasfreshId businessPartnerId;
```

Comment:
is this name linked to some requirement?
if not,  would suggest to call it `bpartnerId` as we have it all over?

@metas-ts  wdyt?